include/build_config/SDL_build_config.h.cmake

```diff
@@ -73,10 +67,7 @@
 /* C library functions */
 #cmakedefine HAVE_DLOPEN 1
 #cmakedefine HAVE_MALLOC 1
-#cmakedefine HAVE_CALLOC 1
-#cmakedefine HAVE_REALLOC 1
```

Comment:
calloc, realloc and free can be removed from the symbols_to_check list in `CMakeLists.txt`.